modules/util.py

```
@@ -543,3 +543,23 @@ def get_image_size_info(image: np.ndarray, aspect_ratios: list) -> str:
+def get_output_folder():
+    import os
+    base = "Outputs"
+    user = get_current_user() or "default"
+    user_path = os.path.join(base, user)
+    
+    if not os.path.exists(user_path):
+        os.makedirs(user_path, exist_ok=True)
+    
+    return user_path
+
+def get_current_user():
```

Ruff (0.8.2)

Redefinition of unused `get_current_user` from line 547

(F811)